includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -264,6 +265,11 @@ public function create_intention(
 		$request['payment_method_types'] = $payment_methods;
 		$request['capture_method']       = $capture_method;
 
+		if ( Buyer_Fingerprinting_Service::is_fraud_prevention_enabled_for_store() ) {
```

Comment:
We'd need a new test case for `create_intention` which ensures the metadata injected when fraup protection is enabled.